nextflow.config

```diff
@@ -32,6 +32,10 @@ params {
     max_multiqc_email_size     = '25.MB'
     multiqc_methods_description = null
 
+    // bedtools options
+    run_bedtools_coverage      = false
+    anno_file                  = null
```

Comment:
I think for consistency (taking the chance to fix all past-James mistakes :grimacing: ), I would rename this parameter to

```
mapstats_bedtools_featurefile = null
```